### ISSUE TITLE
Bugs

### ISSUE THREAD

**Issue by akshatvg** [CodeChefVIT/AnnaRun#3]
1. It should work on small screens as well.
2. Even on desktops, the game isn't on fullscreen yet. It if fixed to the top left for a particular size.
3. The location of powerups, etc should change on page refresh.

**Comment by gvinayakgupta** [comment]
> 1. It should work on small screens as well.
> 2. Even on desktops, the game isn't on fullscreen yet. It if fixed to the top left for a particular size.
> 3. The location of powerups, etc should change on page refresh.

1 & 2 fixed
3 will fix after ghost thingy
3 is basically creating more grid levels, which are still hardcoded.. Will see if it can be automated and if not then create more levels by hardcoding

**Comment by akshatvg** [comment]
Okay. Comment and tag me here once 3 is done.

**Comment by gvinayakgupta** [comment]
@akshatvg new pr me position of food, powerups and pacman is dynamic.. changes on every refresh..
If thats fine.. please close the issue
